test/evaluatex.spec.js

```diff
@@ -144,4 +155,22 @@ describe("Evaluatex", function () {
        const  fn = evaluatex("\\frac 1{20}3", {}, { latex: true });
         fn;
     });
+
+    it("throws error with unset LaTeX symbol", function () {
+      try {
+        // `b` isn't bound
+        test("{a}\\cdot{b}", undefined, { a: 2 }, {}, { latex: true });
+      } catch(err) {
+        assert.equal(true, err instanceof Error);
```

Comment:
This assertion should also check that the error message is correct and helpful, otherwise it passes on any error.

Chai also has an [instanceOf](https://www.chaijs.com/api/assert/#method_assert) assertion that would be more semantic here.